src/domains/transaction/pages/TransactionSend/TransactionSend.tsx

```diff
@@ -200,16 +183,51 @@ export const FifthStep = () => {
 	);
 };
 
-type Props = {
-	onCopy?: () => void;
-	formValues: any;
-};
+export const TransactionSend = () => {
+	const { t } = useTranslation();
 
-export const TransactionSend = ({ onCopy, formValues }: Props) => {
-	const [activeTab, setActiveTab] = React.useState(1);
+	const [activeTab, setActiveTab] = useState(1);
+	const [formData, setFormData] = useState(null);
 	const activeProfile = useActiveProfile();
 
-	const { t } = useTranslation();
+	const form = useForm({ mode: "onChange" });
+
+	const { formState, getValues, register } = form;
+
+	useEffect(() => {
+		register("network", { required: true });
+		register("recipients", { required: true, validate: (value) => Array.isArray(value) && value.length > 0 }); // TODO: min doesn't work
+		register("senderAddress", { required: true });
+		register("fee", { required: true });
+		register("smartbridge");
+	}, []);
+
+	const submitForm = async () => {
+		const { fee, mnemonic, recipients, senderAddress, smartbridge } = getValues();
+		const senderWallet = activeProfile
+			?.wallets()
+			.values()
+			.find((wallet: Wallet) => wallet.address() === getValues("senderAddress"));
+		const transactionId = await senderWallet?.transaction().signTransfer({
+			fee,
+			// nonce: senderWallet?.nonce().plus(1).toString(),
```

Comment:
The nonce should generally be omitted because there is a lot of different ways the nonce has to be handled depending on the coin you are interacting with.

Reply:
No problem, that makes sense. This comment wasn't supposed to be committed 👌